torchvision/prototype/features/_bounding_box.py

```diff
@@ -69,6 +69,16 @@ def to_format(self, format: Union[str, BoundingBoxFormat]) -> BoundingBox:
             self, _F.convert_bounding_box_format(self, old_format=self.format, new_format=format), format=format
         )
 
+    @property
+    def area(self) -> torch.Tensor:
+        if self.format in {BoundingBoxFormat.XYWH, BoundingBoxFormat.CXCYWH}:
+            _, _, width, height = torch.unbind(self, dim=-1)
+        else:  # self.format == BoundingBoxFormat.XYXY
+            x1, y1, x2, y2 = torch.unbind(self, dim=-1)
+            width = x2 - x1
+            height = y2 - y1
+        return width * height
```

Comment:
I don't think we should be implementing an op here. Those `_Feature` objects should be extremely thin wrappers of existing ops. Note there is already an operator for box area estimation called `box_area()`.

BTW what you do here is introduce the idiom of attaching custom methods and properties to specific `_Feature` subclasses but not to all of them. We got a couple of utility methods that do this already (like one for formatting, plotting etc) but it's not clear if we wanna add those methods to the objects. I don't object to that, just flagging it as something "new".

Reply:
> I don't think we should be implementing an op here. Those `_Feature` objects should be extremely thin wrappers of existing ops.

You are right. Let's not introduce new idioms for now.

> Note there is already an operator for box area estimation called `box_area()`.

For reference: 

https://github.com/pytorch/vision/blob/9c3e2bf46bc49997679785d76b7d0a9fea0223c7/torchvision/ops/boxes.py#L219

I'm not going to use that since it would require to always convert to `XYXY` although the other formats are even easier for this computation. 

> BTW what you do here is introduce the idiom of attaching custom methods and properties to specific _Feature subclasses but not to all of them. We got a couple of utility methods that do this already (like one for formatting, plotting etc) but it's not clear if we wanna add those methods to the objects.

I think this is a case that makes sense to have it on the `features.BoundingBox`. Compared to the other kernels this is does not return a somehow changed bounding box, but rather a new quantity. Thus, this should not be a transform.